portfolio/src/main/java/com/google/sps/servlets/DeleteCommentsServlet.java

```diff
@@ -0,0 +1,56 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.servlets;
+ 
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query;
+import com.google.appengine.api.datastore.Query.SortDirection;
+import com.google.appengine.api.datastore.FetchOptions;
+import com.google.gson.Gson;
+import com.google.sps.data.Comment;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+ 
+ 
+/** Servlet responsible for listing comments. */
+@WebServlet("/delete-comments")
+public class DeleteCommentsServlet extends HttpServlet {
+  private DatastoreService datastore; 
+  
+  @Override
+  public void init(){
+    datastore = DatastoreServiceFactory.getDatastoreService();
+  } 
+    
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
```

Comment:
There is an HTTP delete method that is more appropriate to use here (GET, POST, and PUT) are all HTTP methods. More details here: https://developer.mozilla.org/en-US/docs/Web/HTTP/Methods

You should just be able to change this to "doPost" and then set the method to "delete" in your JavaScript.

Reply:
Changing it to "doDelete" and changing the method in the js to "delete" worked but 
"doPost" and the method in js "delete" didn't work

Reply:
Sorry, I meant to type "doDelete" instead of "doPost".